hostplumber/main.go

```diff
@@ -68,6 +113,16 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	if os.Getenv("METRICS_BIND_ADDRESS") != "" {
+		metricsAddr = os.Getenv("METRICS_BIND_ADDRESS")
+	}
+
+	err := validateIPAddressPort(metricsAddr)
+	if err != nil {
+		setupLog.Error(err, "METRICS_BIND_ADDRESS is invalid")
```

Comment:
This message assumes that the bind address is only set using the env var `METRICS_BIND_ADDRESS`. Rather, make it more generic?
```
metrics bind address invalid
```

Reply:
Made the change